workflow-language/src/main/resources/swadl-schema-1.0.json

```diff
@@ -2622,19 +2622,12 @@
         },
         "user-ids-inner": {
             "description": "User identifiers list.",
-            "anyOf": [
```

Comment:
Do you still have auto completion with this?

Reply:
Yes